fusion_toolbox/reaction_rate/bosch_hale.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import sys
+import matplotlib.pyplot as plt
+
+## Class for Bosch-Hale reactivity
+## All of it comes from: H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611
+class Reactivity:
+
+	known_reactions = ["DDn", "DDp", "DTn", "D3Hep"]
+
+	# The space is here so the indexing matches the paper
+	DDn_Cs   = [" ", 5.43360e-12, 5.85778e-3, 7.68222e-3, 0.0, -2.96400e-6, 0.0, 0.0]
+	DDp_Cs   = [" ", 5.65718e-12, 3.41267e-3, 1.99167e-3, 0.0, 1.05060e-5, 0.0, 0.0]
+	DTn_Cs   = [" ", 1.1302e-9, 1.51361e-2, 7.51886e-2, 4.60643e-3, 1.35e-2, -1.0675e-4, 1.366e-5]
+	D3Hep_Cs = [" ", 5.51036e-10, 6.41918e-3, -2.02896e-3, -1.91080e-5, 1.35776e-4, 0.0, 0.0 ]
+
+	DDn_Bg   = 31.3970
+	DDp_Bg   = 31.3970
+	DTn_Bg   = 34.3827
+	D3Hep_Bg = 68.7508
+
+	DDn_mr   = 937814
+	DDp_mr   = 937814
+	DTn_mr   = 1124656
+	D3Hep_mr = 1124572
+
+	Cs_dict = {"DDn" : DDn_Cs, "DDp" : DDp_Cs, "DTn" : DTn_Cs, "D3Hep" : D3Hep_Cs}
+
+	Bg_dict = {"DDn" : DDn_Bg, "DDp" : DDp_Bg, "DTn" : DTn_Bg, "D3Hep" : D3Hep_Bg}
+	mr_dict = {"DDn" : DDn_mr, "DDp" : DDn_mr, "DTn" : DTn_mr, "D3Hep" : D3Hep_mr}
```

Comment:
No whitespace before colons (PEP8)

```suggestion
	Cs_dict = {"DDn": DDn_Cs, "DDp": DDp_Cs, "DTn": DTn_Cs, "D3Hep": D3Hep_Cs}

	Bg_dict = {"DDn": DDn_Bg, "DDp": DDp_Bg, "DTn": DTn_Bg, "D3Hep": D3Hep_Bg}
	mr_dict = {"DDn": DDn_mr, "DDp": DDn_mr, "DTn": DTn_mr, "D3Hep": D3Hep_mr}
```